io/disk/mvcli_fw_flash.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2017 IBM
+# Author: Pridhiviraj Paidipeddi <ppaidipe@linux.vnet.ibm.com>
+
+"""
+Mvcli Firmware Flash Test for OpenPower based Marvell adapter 88SE9230.
+"""
+
+import shutil
+import platform
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.process import CmdError
+
+
+class MvcliFwFlash(Test):
```

Comment:
Please have a look at nvmetest.py on how firmware flash is handled in the same test as other tests